src/main/java/io/lighty/yang/validator/formats/yang/printer/ModulePrinter.java

```diff
@@ -109,22 +109,22 @@ private void printAugmentations() {
         for (final AugmentationSchemaNode augmentation : module.getAugmentations()) {
             boolean printOpeningStatement = true;
             for (SchemaTree st : schemaTree) {
-                if (st.isAugmenting()
-                        && st.getSchemaNode().getPath().getParent()
-                        .equals(augmentation.getTargetPath().asSchemaPath())) {
-                    if (printOpeningStatement) {
-                        final StringBuilder target = new StringBuilder();
-                        for (final QName name : augmentation.getTargetPath().getNodeIdentifiers()) {
-                            target.append('/');
-                            target.append(moduleToPrefix.get(name.getModule()));
-                            target.append(':');
-                            target.append(name.getLocalName());
-                        }
-                        printer.openStatement(Statement.AUGMENT, target.toString());
-                        printOpeningStatement = false;
+                if (isStNotAugmentOrStParentNotEqualsToAugmPath(st, augmentation)) {
```

Comment:
use positive name for method

that brings possibility to remove `continue` branch

Reply:
done